### ISSUE TITLE
Best practice for assigning api parameter?

### ISSUE THREAD

**Issue by xrx101** [browser-update/browser-update#525]
When customising the service, the api version is specified like so:

`api: xxx   // This is the version of the browser-update api to use. Please do not remove.`

In the sample code to copy and paste, the api version is specified as a decimal number like this ==> api: 2021.03
At the time of this writing,  it is 2021/03/21, so it makes sense to me that this will need to be automatically incremented for the current yyyy.mm?  

Seeking clarity on how it is actually implemented behind the scenes because I've put 2021.10 and 2000.10 - nothing different seems to happen.  I've peeked into the source and I see it is appended to the source: 
```
var extra=encodeURIComponent("frac="+frac+"&txt="+txt+"&apiver="+op.apiver);
i.src="https://browser-update.org/cnt?what=noti&from="+bb.n+"&fromv="+bb.v + "&ref="+ escape(op.pageurl) + "&jsv="+op.jsv+"&tv="+op.style+"&extra="+extra;
```
Bottom line, is it recommended that the current date in the format of yyyy.mm is used so as to not hardcode it to a specific version? 
Or simply omit it as default would be the latest?  Given the comment "Please do not remove", it doesn't seem likely that it would default to anything.

Guidance appreciated.

**Comment by josselex** [comment]
Hi,
You should ad the date the code was created. 
It is just in case there are some backward incomaptible changes to the configuration or default values in the future. 
Then I can look what API version the people were using and update the script without breaking the sites' intended configuration.

Does this help?

**Comment by xrx101** [comment]
Ah, okay.  For some reason I was thinking that in order for the parameters of browser versions to work, it would use the api version.   Good to know that isn't the case and makes sense what you have said.  Thanks.
